src/etc/utils.ts

```diff
@@ -23,109 +18,80 @@ const pack = fs.readJSONSync(packagePath);
  * @returns The package version
  * @todo grab potential labels from the version and provide them as further pegs in the menu.
  */
-export function getSemanticVersion(
-	version: string | null = pack.version
-): patchVersion {
+export function getSemanticVersion(version: string = pack.version): version {
```

Comment:
I prefer to leave the `version: string | null = pack.version` so that we can get test coverage on the error handler.

Reply:
The issue with leaving the `| null` is that it can mislead you into thinking that passing `null` is an acceptable use of the function. If the intention is to allow for testing the thrown Error, you can do that without allowing `null` in TS:

```ts
               it('throws error if version not defined', function () {
			assert.throws(() => {
				// @ts-expect-error: Intentionally passing a falsy value to test the error condition.
				vUtils.getSemanticVersion(null);
			}, 'Package version was not found');
		});
```
In the above, the `@ts-expect-error` tells TS that we know what we're doing, and to ignore the fact that `null` is not a string and this will definitely result in an error, and the tests pass as expected :)

Reply:
Resolved by 148563f91a206b97b60f4642f2dd985b0acced72